broskichat-app/src/Components/Register/Register.jsx

```diff
@@ -91,25 +91,45 @@ class Register extends React.Component {
     event.preventDefault();
   }
 
+  // Style the horizontal line (This may be moved to shared);
```

Comment:
my general rule is once we use something twice, we're probably going to use it three times and we should component-ify it